app/templates/subsidy/measure-offer.hbs

```diff
@@ -43,13 +43,7 @@
                 @route="subsidy.application-steps"
                 @model={{series.id}}
               >
-                {{#if series.title}}
-                  {{series.title}}
-                {{else}}
-                  <AuHelpText>
-                    Geen titel
-                  </AuHelpText>
-                {{/if}} 
+                {{series.label}}
```

Comment:
@ZijneDwaasheid I changed this to use the label function, which checks for title and fallbacks to create title based on start/end date of period. It's also used in loket so best to mimic closely